src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -289,10 +278,10 @@ private void PendBindingInformation(ConnectionInformation connInfo)
 
         private void UpdateSolutionFile()
         {
-            foreach (ConfigFileInformation info in bindingConfigInformationMap.Values)
+            foreach (var info in bindingConfigInformationMap.Values)
             {
-                Debug.Assert(fileSystem.File.Exists(info.NewFilePath), "File not written " + info.NewFilePath);
-                legacyConfigFolderItemAdder.AddToFolder(info.NewFilePath);
+                Debug.Assert(fileSystem.File.Exists(info.FilePath), "File not written " + info.FilePath);
+                legacyConfigFolderItemAdder.AddToFolder(info.FilePath);
```

Comment:
Hhhmm. This will complicate removing `FilePath` from the interface.

Reply:
Yep :(